synapse/cortex.py

```diff
@@ -403,48 +412,62 @@ def getCoreInfo(self):
         Yields:
             ((str,dict)): Storm messages.
         '''
-        view = self.cell._viewFromOpts(opts, self.user)
+        opts = self._reqValidStormOpts(opts)
 
         if opts is not None and opts.get('spawn'):
+            await self._execSpawnStorm(text, opts)
```

Comment:
the precondition above for ``opts is not None`` is no longer true with `_reqValidStormOpts` and can be removed.